hoomd/hpmc/ComputeSDF.h

```diff
@@ -264,27 +269,64 @@ template<class Shape> void ComputeSDF<Shape>::zeroHistogram()
     // Zero the histogram
     for (size_t i = 0; i < m_hist.size(); i++)
         {
-        m_hist[i] = 0;
+        m_hist[i] = 0.0;
         }
     }
 
+template<class Shape> Scalar ComputeSDF<Shape>::getMaxInteractionDiameter()
+    {
+    Scalar max_core_diameter = m_mc->getMaxCoreDiameter();
+    Scalar max_r_cut_patch(0.0);
```

Comment:
```suggestion
    Scalar max_r_cut_patch = 0.0;
```
We don't normally use constructors when initializing basic types.